pom.xml

```diff
@@ -1590,9 +1590,9 @@
         <sandesha2.version>1.6.1-wso2v1</sandesha2.version>
         <geronimo.specs.jms.version>1.1.1</geronimo.specs.jms.version>
         <httpcore-nio.wso2.version>4.3.3.wso2v4</httpcore-nio.wso2.version>
-        <rampart.wso2.version>1.6.1-wso2v26</rampart.wso2.version>
+        <rampart.wso2.version>1.6.1-wso2v29</rampart.wso2.version>
         <neethi.orbit.version>2.0.4.wso2v4</neethi.orbit.version>
-        <rampart.mar.version>1.6.1-wso2v26</rampart.mar.version>
+        <rampart.mar.version>1.6.1-wso2v29</rampart.mar.version>
```

Comment:
let's make this refer to rampart.wso2.version, that way we can avoid having to modify the version at multiple places

Reply:
done